Tiltfile

```diff
@@ -208,8 +283,7 @@ def cspo_template():
     cspo_yaml = local(cmd, quiet=True)
     k8s_yaml(cspo_yaml)
     k8s_resource(objects = ["cspotemplate:openstackclusterstackreleasetemplate"], new_name = "cspotemplate", labels = ["cspo-template"])
-
-
+    
 def clusterstack():
     k8s_resource(objects = ["clusterstack:clusterstack"], new_name = "clusterstack", labels = ["clusterstack"])
```

Comment:
I don't know why, but in the local mode this clusterstack is red and needs retrigger

Reply:
Yes, I was not able to find a good solution for it. It's happening because we are applying clusterstack too early before the CRDs are applied.

Reply:
Did you try the same way as cspo_template() is done?

Reply:
Roman, I didn't gave much attention to this one. 
I created an issue here https://github.com/SovereignCloudStack/cluster-stack-provider-openstack/issues/91 

Let's do it in a separate PR, this can be handled separately. This is not a blocker for me as of now in using tilt setup. I'll do a separate PR for this.

Reply:
Thank you!

Reply:
During testing, I see cspotemplate again red and needs to be triggered, can you add that to issue 91 as well, please?

Reply:
Maybe after adding `openstackclusterstackreleasetemplates` to k8s_resource objects it happened? I don't know

Reply:
Yes, I noticed it happened again but we are going to apply the templates in the subsequent step after the controller starts. 
Tilt will only start the controller and then we apply the resources using a click that will be backed by a make target.